README.md

```diff
@@ -45,4 +45,10 @@ It's also possible to use in your own Dockerfile:
 
     FROM prom/mesos-exporter
 
+
+## Console Dashboard
+
+Copy the content of `consoles` to the consoles folder used by your Prometheus master. Your Mesos
```

Comment:
These currently live in the prometheus repo, as there's more changes than this required for it to hook into the menu and table.